servicebus-parametrised.tf

```diff
@@ -1,13 +1,13 @@
-module "servicebus-namespace-premium" {
```

Comment:
you need to use a `moved` block to rename the terraform resource without deleting

Reply:
think this is sorted now